tests/unittests/config/test_cc_package_update_upgrade_install.py

```diff
@@ -156,7 +156,7 @@ def _new_subp(*args, **kwargs):
         assert m_subp.call_args_list[0] == mock.call(["apt-cache", "pkgnames"])
 
         for arg in ["apt-get", "install", "pkg1", "pkg2"]:
-            assert arg in m_subp.call_args_list[1][1]["args"]
+            assert arg in m_subp.call_args_list[1][0][0]["args"]
```

Comment:
What has changed about the args list in these tests? The three similar changes in this file all break tests for me.

Reply:
I think this is one of the tests that's actually failing currently.